kitsune/wiki/jinja2/wiki/includes/sidebar_modules.html

```diff
@@ -24,9 +24,14 @@ <h3 class="sidebar-subheading large-only">{{ _('Editing Tools') }}</h3>
             </li>
           {% endif %}
           {% if document and not document.is_archived and (document.allows(user, 'create_revision') or document.allows(user, 'edit')) and fallback_reason != "fallback_locale" %}
-            <li {{ active|class_selected('edit') }}>
-              <a href="{{ url('wiki.edit_document', document.slug) }}">{{ _('Edit Article') }}</a>
+          <li {{ active|class_selected('edit') }}>
+            <a href="{{ url('wiki.edit_document', document.slug) }}">{{ _('Edit Article') }}</a>
+          </li>
+            {% if request.user.is_staff %}
```

Comment:
If you are a `superuser` you are automatically given the `staff` role. Since someone in the `administrator` group could not be staff, in order to do what was asked, I'm only looking for staff.